core/driver/sources/src/main/scala/org/virtuslab/ideprobe/dependencies/Resource.scala

```diff
@@ -86,6 +87,24 @@ object Resource extends ConfigFormat {
     }
   }
 
+  final class DMGFile(val path: Path) extends IntellijResource {
+    def installTo(target: Path): Unit = {
+      import sys.process._
+
+      // attach disk image from .dmg file to local filesystem
+      val dmgDir = s"${System.getProperty("java.io.tmpdir")}dmg_dir"
+      s"mkdir $dmgDir".!
+      s"hdiutil attach -mountpoint $dmgDir ${path.toString}".!
+
+      // copy $dmgDir/IntelliJ IDEA CE.app to proper installation directory
+      val idePath = Paths.get(s"$dmgDir/IntelliJ IDEA CE.app/Contents")
+      idePath.copyDir(target)
+
+      // detach disk image from local filesystem
+      s"hdiutil detach $dmgDir".!
```

Comment:
Awww okay, nvm ;D

Reply:
Still, I'd use some mechanism like try-with-resources (or just a good old `try`-`finally`) to make sure that dmg is detached even if the copy fails for whatever reason

Reply:
Ok, try-finally added